File: src/main/java/com/krzywus/bkl4/model/Transfer.java
package com.krzywus.bkl4.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Transfer {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    private int transferId;

    @Column(name = "username")
    private String username;

    @Column(name = "recipient")
    private String recipient;

    @Column(name = "account")
    private String account;

    @Column(name = "amount")
    private String amount;

    @Column(name = "title")
    private String title;

}

File: src/ng/src/app/model/transfer.ts
export class Transfer {
    id?: number;
    recipient: string;
    account: string;
    amount: string;
    title: string;


    constructor(recipient: string, account: string, amount: string, title: string, id?:number) {
        this.id = id;
        this.recipient = recipient;
        this.account = account;
        this.amount = amount;
        this.title = title;
    }
}